convene-web/app/models/utility_connection.rb

```diff
@@ -0,0 +1,24 @@
+# Links a {Utilities::Utility} to a {Space} or {Person}.
+class UtilityConnection < ApplicationRecord
```

Comment:
Ok, this is silly, but hear me out: if it is a "utility" in the gas, water, and electricity sense, then this is class is really a "hook up". (I told you this was silly.)

Reply:
No that's amazing I love it.